src/main/java/org/spongepowered/common/command/manager/SpongeCommandManager.java

```diff
@@ -524,6 +526,15 @@ public void init() {
         } catch (final CommandFailedRegistrationException ex) {
             throw new RuntimeException("Failed to create root Sponge command!", ex);
         }
+        try {
+            SpongeParameterizedCommandRegistrar.INSTANCE.register(
+                    Launcher.getInstance().getCommonPlugin(),
+                    ((SpongePaginationService) Sponge.getServiceProvider().paginationService()).createPaginationCommand(),
```

Comment:
This should only run if the service wasn't replaced, this will throw a class cast exception otherwise.

Reply:
Oh right, I assumed it can't be replaced for some reason.